hack/lib/golang.sh

```diff
@@ -68,6 +81,20 @@ else
   fi
   readonly KUBE_SERVER_PLATFORMS
 
+  # The node platforms we build for
+  KUBE_NODE_PLATFORMS=(
+    linux/amd64
+    linux/arm
+    linux/arm64
+    linux/s390x
+    windows/amd64
+    windows/386
```

Comment:
Removed. Windows 2016 is required for Windows containers, and it's not build for 386 afaik know.